packages/gafl-webapp-service/src/processors/licence-type-display.js

```diff
@@ -6,15 +6,15 @@ export const licenceTypeDisplay = (permission, mssgs) => {
 
   // Build the display string for the licence type
   if (concessionHelper.hasJunior(permission)) {
-    typesStrArr.push(mssgs.age_junior)
+    typesStrArr.push(mssgs.junior)
```

Comment:
You've now removed the single usage of `age_junior` and replaced it with `junior`. Please can you change the age_junior label rather than creating a new label.

Reply:
my bad 😓 all fixed 🔨